.circleci/config.yml

```diff
@@ -1,7 +1,6 @@
 version: 2
-
 jobs:
-  "golang-1.14":
+  golang-1.14:
```

Comment:
It looks like this is just a formatting change, but I wonder if we should be updating these? The supported Go releases at this moment in time are Go 1.16, and Go 1.17. I'm not sure we should try to keep this working on older releases?

... although this should probably be a separate change.

Reply:
fixed